packages/dashboard/src/portal/components/VotePopup.vue

```diff
@@ -1,4 +1,4 @@
-<template>
+<template v-if="nodes">
```

Comment:
not sure if the `v-if` works with list length or not, can you make sure?

Reply:
Checked & it works.